templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>
```

Comment:
The `href` attributes are currently set to `#`, which means these links won't navigate to any page. Replace `#` with actual URLs or use Django's `{% url %}` template tag to link to the appropriate views.